clang/test/lab1/kostanyan_arsen/tests.cpp

```diff
@@ -0,0 +1,36 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/PrintNamesPlugin%pluginext -plugin classprinter %s 1>&1 | FileCheck %s
+
+
+class Test1 {
+  int x;
+  double y;
+};
+// CHECK: Test1
+// CHECK-NEXT: |_x
+// CHECK-NEXT: |_y
+
+struct Test2 {
+  float a;
+  char b;
+  bool c;
+};
+// CHECK: Test2
+// CHECK-NEXT: |_a
+// CHECK-NEXT: |_b
+// CHECK-NEXT: |_c
+
+template <typename T> class Test3 { T var; };
+// CHECK: Test3
+// CHECK-NEXT: |_var
+
+class Test4 {
+  class Nested {
+    int z;
+  };
+};
+// CHECK: Test4
+// CHECK: Nested
+// CHECK-NEXT: |_z
+
+// RUN: %clang_cc1 -load %llvmshlibdir/PrintNamesPlugin%pluginext -plugin classprinter -plugin-arg-classprinter --help %s 2>&1 | FileCheck %s --check-prefix=HELP
+// HELP: This plugin traverses the Abstract Syntax Tree (AST) of a codebase and prints the name and fields of each class it encounters
```

Comment:
Please add a `HELP-NOT: |_` check if `help` is specified.

Reply:
@m-ly4 Sorry, i am not sure what i need to do. I assume i have to add `return false` in ParseArgs in order to add `HELP-NOT: |_ ` check, but it gives me Exit Code: 1 when i add `return false`. Could you please explain in more detail what I need to do?

Reply:
You may execute `not %clang_cc1` when it returns 1 but it is expected.